cmd/controller-gen/main.go

```diff
@@ -22,7 +22,7 @@ import (
 	"path/filepath"
 
 	"github.com/spf13/cobra"
-	crdgenerator "sigs.k8s.io/controller-tools/pkg/crd/generator"
+	"sigs.k8s.io/controller-tools/pkg/generate/crd"
```

Comment:
`generate` is not a good pkg name as per Go guidelines. Given that we have moved scaffolding pkgs to `kubebuilder` repo, it will be safe to assume that the pkgs that live here are related to generation. So may be move the others `rbac` and `webhook` under `pkg` ?

Reply:
Agree. So, I will move all other packages, like `rbac`, `webhook` into `pkg` in this PR.